CMakePresets.json

```diff
@@ -0,0 +1,51 @@
+{
+  "version": 10,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 19,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "ios",
+      "displayName": "iOS",
+      "generator": "Xcode",
+      "description": "Create Xcode project for iOS",
+      "binaryDir": "${sourceDir}/build-ios",
+      "cacheVariables": {
+        "CMAKE_SYSTEM_NAME": "iOS",
+        "CMAKE_CXX_COMPILER_LAUNCHER": "ccache",
+        "MLN_WITH_METAL": "ON",
+        "MLN_WITH_OPENGL": "OFF",
+        "MLN_DRAWABLE_RENDERER": "ON",
+        "MLN_WITH_RTTI": "ON"
```

Comment:
I get link errors when I don't enable RTTI...

<img width="402" alt="Screenshot 2025-02-23 at 00 46 22" src="https://github.com/user-attachments/assets/c27a9ca2-702a-4cfa-a90d-55e5ea743338" />

Reply:
I think we'd lose access to `dynamic_cast` if we disable RTTI

Reply:
Fixed by making sure RTTI is disabled for all translation units.